packages/cli/src/install.ts

```diff
@@ -253,7 +292,10 @@ Your token needs these permissions:
         }
     }
 
-    if (await promptDeploy(serverPkgJson.version)) {
+    // Auto-deploy when there's only one account, otherwise prompt
```

Comment:
In hindsight this was a behavior change that I missed – the script always prompted for deploy, 1 account or no.